src/ExpoMessage.php

```diff
@@ -0,0 +1,218 @@
+     * Specify zero to clear the badge.
+     *
+     * @var int
+     */
+    protected $badge = 0;
+
+    /**
+     * The number of seconds for which the message may be kept around for redelivery if it has not been delivered yet.
+     *
+     * @var int
+     */
+    protected $ttl = 0;
+
+    /**
+     * ID of the Notification Channel through which to display this notification on Android devices.
+     *
+     * @var string
+     */
+    protected $channelId = 'Default';
+
+    /**
+     * The json data attached to the message.
+     *
+     * @var string
+     */
+    protected $jsonData = '{}';
+
+    /**
+     * Create a message with given body.
+     *
+     * @param string $body
+     *
+     * @return static
+     */
+    public static function create($body = '')
+    {
+        return new static($body);
+    }
+
+    /**
+     * ExpoMessage constructor.
+     *
+     * @param string $body
+     */
+    public function __construct(string $body = '')
+    {
+        $this->body = $body;
+    }
+
+    /**
+     * Set the message title.
+     *
+     * @param string $value
+     *
+     * @return $this
+     */
+    public function title(string $value)
+    {
+        $this->title = $value;
+
+        return $this;
+    }
+
+    /**
+     * Set the message body.
+     *
+     * @param string $value
+     *
+     * @return $this
+     */
+    public function body(string $value)
+    {
+        $this->body = $value;
+
+        return $this;
+    }
+
+    /**
+     * Enable the message sound.
+     *
+     * @return $this
+     */
+    public function enableSound()
+    {
+        $this->sound = 'default';
+
+        return $this;
+    }
+
+    /**
+     * Disable the message sound.
+     *
+     * @return $this
+     */
+    public function disableSound()
+    {
+        $this->sound = null;
+
+        return $this;
+    }
+
+    /**
+     * Set the message badge (iOS).
+     *
+     * @param int $value
+     *
+     * @return $this
+     */
+    public function badge(int $value)
+    {
+        $this->badge = $value;
+
+        return $this;
+    }
+
+    /**
+     * Set the time to live of the notification.
+     *
+     * @param int $ttl
+     *
+     * @return $this
+     */
+    public function setTtl(int $ttl)
+    {
+        $this->ttl = $ttl;
+
+        return $this;
+    }
+
+    /**
+     * Set the channelId of the notification for Android devices.
+     *
+     * @param string $channelId
+     *
+     * @return $this
+     */
+    public function setChannelId(string $channelId)
+    {
+        $this->channelId = $channelId;
+
+        return $this;
+    }
+
+    /**
+     * Set the json Data attached to the message.
+     *
+     * @param array|string $data
+     *
+     * @return $this
+     *
+     * @throws CouldNotCreateMessage
+     */
+    public function setJsonData($data)
+    {
+        if (is_array($data)) {
+            $data = json_encode($data);
+        } elseif (is_string($data)) {
+            @json_decode($data);
```

Comment:
Can we remove the error suppression operator?
`json_decode` doesn't throw an exception